can/io/logger.py

```diff
@@ -217,8 +217,8 @@ def _get_new_writer(self, filename: StringPathLike) -> FileIOMessageWriter:
             An instance of a writer class.
         """
         # Close the old writer first
-        if self._writer is not None:
-            self._writer.stop()
+        # if self._writer is not None:
+        #     self._writer.stop()
```

Comment:
Commenting these lines out stops the error message from being thrown. The rollover process works properly. Untested with other file formats.

Reply:
Looks good to me. Remove the comment and adapt the docstring of the method. Then i'll approve this.